shopfloor_reception_shipment_advice/services/reception.py

```diff
@@ -34,22 +69,22 @@ def _scan_document__by_shipment(self, shipment, barcode):
             return self._response_for_select_document(
                 message=self.msg_store.shipment_nothing_to_unload(shipment)
             )
-
-        return self._response_for_select_move(picking=pickings, shipment=shipment)
+        self.work.current_shipment_advice = shipment
+        return self._response_for_select_move(picking=pickings)
 
     def _scan_document__get_handlers_by_type(self):
         res = super()._scan_document__get_handlers_by_type()
         res["shipment"] = self._scan_document__by_shipment
         res["dock"] = self._scan_document__by_dock
         return res
 
-    def _response_for_select_move_get_data(self, picking, **kwargs):
+    def _response_for_select_move_get_data(self, picking):
         data = {}
         if picking:
             data = super()._response_for_select_move_get_data(picking)
-        if "shipment" in kwargs.keys():
+        if self.shipment_advice:
```

Comment:
isn't this supposed to match this attribute definition?
https://github.com/OCA/wms/pull/757/commits/b20b551eba3c1da68df09e6ac7f773b83942fcd6#diff-ccbdac8a372d07e032a6389742df2f833901754054c6898ce8e7e6072f484554R72